spec/features/schools/participants/transferring_participants/fip_to_fip/ect_no_change_spec.rb

```diff
@@ -0,0 +1,361 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe "transferring participants", with_feature_flags: { change_of_circumstances: "active" }, type: :feature, js: true, rutabaga: false do
+  context "Transferring an ECT to a school" do
+    context "ECT has matching lead provider and delivery" do
+      before do
```

Comment:
Minor - missing 'parter'? e.g. ECT has matching lead provider and delivery partner